crates/control/src/behavior/calibrate.rs

```diff
@@ -0,0 +1,11 @@
+use types::{MotionCommand, PrimaryState, WorldState};
+
+pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
+    match world_state.robot.primary_state {
+        PrimaryState::Calibration => Some(MotionCommand::Stand {
+            head: types::HeadMotion::Unstiff,
+            is_energy_saving: false,
```

Comment:
Request for comments: Should we enable energy-saving stand? Is this already implemented?

Reply:
It is implemented, but disabled by default, see parameter `energy_saving_stand.enabled`.
Not sure how much of a difference that makes for calibration, but the head will be about 3 cm higher above the ground with energy saving stand enabled.

Reply:
Ok, let's don't care then and keep it as it is. :muscle: